reframe/frontend/reporting/storage.py

```diff
@@ -233,26 +233,39 @@ def _decode_sessions(self, results, sess_filter):
 
         Return a map of session uuids to decoded session data
         '''
+        sess_info_patt = re.compile(
+            r'\"session_info\":\s+(?P<sess_info>\{.*?\})'
+        )
+
+        def _extract_sess_info(s):
+            return sess_info_patt.search(s).group('sess_info')
+
+        @time_function
+        def _mass_json_decode(json_objs):
+            data = '[' + ','.join(json_objs) + ']'
+            getlogger().debug(f'decoding {len(data)} bytes')
```

Comment:
```suggestion
            getlogger().debug(f'decoding {len(data)} code points')
```

Reply:
Yes, they are not bytes :-) But few will understand the code points seeing the logs. I would also add the byte count (assuming ASCII characters which is 99% of the cases). Something like:

```
decoding JSON raw data of length X (X bytes assuming ASCII chars)
```